### ISSUE TITLE
I cannot see my solutions for solved katas.

### ISSUE THREAD

**Issue by guangchl** [Codewars/codewars.com#274]
Today I finished several katas, however, I cannot see them now. I can still check my solutions I made on the day when I signed up.

**Comment by luther9** [comment]
Is it possible that we have to hit the Submit button twice to have our solution recorded?

**Comment by guangchl** [comment]
@luther9 Good catch. Just tried again, double hit do help even though the delay could always be there.

**Comment by luther9** [comment]
It still doesn't work reliably. After re-doing my katas yesterday, I can still only see 3 out of 4 of my solutions today.

**Comment by guangchl** [comment]
@luther9 That's not too bad though. Sometimes I triple click the submit and get 2 identical submissions. So I think, always double click and definitely same the solution for easier copy paste later.

**Comment by azzamsa** [comment]
I get this problem too, today.

**Comment by azzamsa** [comment]
I can't find my solution even after re-doing and re-submiting.

**Comment by azzamsa** [comment]
if someone encounter the same problem, try use 'codewars.com/users/yourname/completed_solutions'

**Comment by taorkon** [comment]
I seem to have run into this issue as well. I have no access to others' solutions (I still see mine though)
![Screenshot from 2023-09-23 01-24-43](https://github.com/codewars/codewars.com/assets/27294801/60059aea-f346-4ea6-8ccc-b1b34bab545e)
